system/processes/detectors-processes.tf

```
@@ -0,0 +1,30 @@
+resource "signalfx_detector" "processes" {
+  name = "${join("", formatlist("[%s]", var.prefixes))}[${var.environment}] Processes aliveness"
+
+  program_text = <<-EOF
+        signal = data('ps_count.processes', filter=${module.filter-tags.filter_custom})${var.processes_aggregation_function}.${var.processes_transformation_function}(over='${var.processes_transformation_window}').publish('signal')
+        detect(when(signal < ${var.processes_threshold_critical})).publish('CRIT')
+        detect(when((signal < ${var.processes_threshold_warning}) and (signal > ${var.processes_threshold_critical}))).publish('WARN')
+  EOF
+
+  rule {
+    description           = "is too low < ${var.processes_threshold_critical}"
```

```suggestion
    description           = "count is too low < ${var.processes_threshold_critical}"
```

"processes count is" or "processes are", as you prefer

---

Change done